### ISSUE TITLE
Popups and new tabs are opened in the browser

### ISSUE THREAD

**Issue by Variann** [Ellpeck/ObsidianCustomFrames#40]
Currently I'm using a custom pane with Google Drive on it to see all my documents, problem is; Drive forces all files you open to be opened in a new tab and I guess CustomFrames can't create new panes on the fly for new tabs, so it just opens it in your browser.
I would like panes to just simply navigate to the new link instead of attempting to open it in my browser.

**Comment by schloo** [comment]
I have the same problem as #73 -- As you suggested, I tried google.com and it still opened the login page in a new tab. I did finally find that myaccount.google.com loads a login page inline, but it threw a 400 error after that so I still haven't been able to login successfully.

**Comment by FeiLZX3Km** [comment]
> Currently I'm using a custom pane with Google Drive on it to see all my documents, problem is; Drive forces all files you open to be opened in a new tab and I guess CustomFrames can't create new panes on the fly for new tabs, so it just opens it in your browser. I would like panes to just simply navigate to the new link instead of attempting to open it in my browser.

Yes, I need this feature too, and I hope developers will prioritize it. Labor driven! Thank you~

**Comment by 712346867** [comment]
The surfing plugin made it easy to solve the issue, but finding that solution was quite a challenge. It took me 4 hours to figure it out.
Initially, I logged into ChatGPT using my Google account, but I have two accounts and couldn't switch between them successfully. I tried creating a new Obsidian vault, deleting and re-adding the website, but none of these methods worked to change the logged-in Google account.
CustomFrames cannot switch Google accounts, probably because it can't open pop-ups. The pop-up keeps calling Chrome, and there's no API connection between Obsidian and Chrome.

**Comment by Ellpeck** [comment]
This issue has now been resolved fully with the release of the web viewer core plugin and the companion fix #136! For more info, check out the known issues section of the readme: https://github.com/Ellpeck/ObsidianCustomFrames/blob/master/README.md#%EF%B8%8F-known-issues
